{"uuid": "10a3ac2b-1e29-417a-85d2-ee4e8acf20b1", "children": ["b1823b27-2c4b-4fdb-8c33-422f164783df", "bcce4fc1-3a1a-47ca-8c61-5b93e4ddfd93", "30e9c026-837b-4215-bbc5-5fa0e8dfe67a", "5b7c9878-8bdb-4309-b05c-5cf2b22dd210", "cba1e572-d342-4711-95fa-43dc3991250b", "e4e68370-30eb-4647-9a68-09d1801be287"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408536, "stop": 1699988408536}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408545, "stop": 1699988408545}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408551, "stop": 1699988408551}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408557, "stop": 1699988408557}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408563, "stop": 1699988408563}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988408569, "stop": 1699988408569}], "start": 1699988408569, "stop": 1699988408573}